### PR TITLE
refactor(flex): Refine the schema definition of `rt_mutable_graph`

### LINKED ISSUE #3077: [Flex] Refine graph schema definition
Current graph schema definition for `rt_mutable_graph` can be revised into a more comprehensive version.

### PR DESCRIPTION
**Pull request by zhanglei1949**
## What do these changes do?

- Adapt to a schema definition, change `modern_graph.yaml`, `bulk_load.yaml`.
- Introduce new test to verify graph loading.
- All csr related params are put in `x_csr_params`
- Our schema definition support each label of Vertex has at least one primary key. But current implementation doesn't support Primary keys(Maybe supported later).

Other repos reflected by this PR: gstest, flex_ldbc_snb

## Related issue number

Fixes #3077

### CODE REVIEW

File: flex/storages/rt_mutable_graph/schema.cc [hunk 1]
```diff
@@ -646,63 +814,94 @@ static bool parse_edge_files(
       std::string file_format;
       std::string file_path;
       if (!get_scalar(files_node[i], "format", file_format)) {
+        LOG(ERROR) << "format is not set properly";
         return false;
       }
       if (file_format != "standard_csv") {
-        LOG(INFO) << "file_format is not set properly";
+        LOG(ERROR) << "file_format is not set properly";
```
Comment (longbinlai) [inline flex/storages/rt_mutable_graph/schema.cc:821]: typically, in the error info, we should also output the provided `file_format`, and also give hints about supported one.

File: flex/tests/rt_mutable_graph/test_graph_loading.cc [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  data_path = argv[3];
+
+  double t0 = -grape::GetCurrentTime();
+  auto& db = gs::GraphDB::get();
+
+  auto ret = gs::Schema::LoadFromYaml(schema_file, bulk_load_config_path);
+  db.Init(std::get<0>(ret), std::get<1>(ret), std::get<2>(ret),
+          std::get<3>(ret), std::get<4>(ret), data_path, 1);
+
+  t0 += grape::GetCurrentTime();
```
Comment (longbinlai) [inline flex/tests/rt_mutable_graph/test_graph_loading.cc:42]: Can we remove this unnecessary grape dependencies?

File: flex/storages/rt_mutable_graph/mutable_property_fragment.h [hunk 3]
```diff
@@ -108,7 +108,7 @@ class MutablePropertyFragment {
   void initEdges(
       label_t src_label_i, label_t dst_label_i, label_t edge_label_i,
       const std::vector<std::tuple<std::string, std::string, std::string,
```
Comment (longbinlai) [inline flex/storages/rt_mutable_graph/mutable_property_fragment.h:110]: 这种超长的类型建议额外定义一下。std::tuple<xxxxx> 这个，我看蛮多地方用的。

File: flex/tests/rt_mutable_graph/test_graph_loading.cc (hunk 2 above)
Comment (zhanglei1949) [reply flex/tests/rt_mutable_graph/test_graph_loading.cc:42]: [An issue](https://github.com/alibaba/GraphScope/issues/3094) is created and I will work on reducing dependencies and reducing image size.

File: flex/storages/rt_mutable_graph/mutable_property_fragment.h (hunk 3 above)
Comment (zhanglei1949) [reply flex/storages/rt_mutable_graph/mutable_property_fragment.h]: Introduce VertexLoadingMeta and EdgeLoadingMeta to avoid explicit tuple.

**Review by longbinlai — DISMISSED** [review]

**Review by luoxiaojian — APPROVED** [review]

**Review by longbinlai — APPROVED** [review]

**Review by luoxiaojian — APPROVED** [review]

**Review by longbinlai — APPROVED** [review]

**Pull request merged**